src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,26 @@
 
   @include hidden;
 
-  bottom: 2em;
+  bottom: 0;
   font-size: 1.2em;
-  left: 3em;
+  left: 0;
   pointer-events: none;
   position: absolute;
-  right: 3em;
+  right: 0;
   text-align: center;
   transition: bottom $animation-duration-short ease-out;
+  top: 0;
+
+  .#{$prefix}-subtitle-position-default {
```

Comment:
Shouldn't that be nested via `&.#{$prefix}-subtitle-position-default {` into the `.#{$prefix}-subtitle-region-container` 🤔 Does it make sense to have this class on something else than a container 🤔